pages/guide/08-start.md

```diff
@@ -0,0 +1,422 @@
+---
+layout: guide
+group: guide
+title: How to Semantic Model Guide
+---
+
+{:toc}
+
+### How do I get the demo app running on my machine using the default semantic model provided?
+
+Check the [Quick Start Guide](https://yavin.dev/pages/guide/02-start.html)
+
+For the sample demos that comes with Yavin (we are using the [Netflix Movie and TV Shows](https://www.kaggle.com/shivamb/netflix-shows) dataset that is sourced from [Kaggle](https://www.kaggle.com/) :
+
+[The DB connection config](https://github.com/yahoo/yavin/blob/master/packages/webservice/app/src/main/resources/demo-configs/db/sql/DemoConnection.hjson)
+[The default semantic model config](https://github.com/yahoo/yavin/blob/master/packages/webservice/app/src/main/resources/demo-configs/models/tables/DemoTables.hjson)
+
+### How do I connect to a new database?
+
+In path : ```yavin/packages/webservice/app/src/main/resources/demo-configs/db/sql/```
```

Comment:
Feel free to ignore. The flow of the sentence seems off. 

`Create a Hjson file for each data connection in path 
```yavin/packages/webservice/app/src/main/resources/demo-configs/db/sql/```
`